Vagrantfile

```diff
@@ -52,7 +54,7 @@ Vagrant.configure(2) do |config|
     end
 
 
-    controlplane.vm.provision :shell, privileged: false, inline: "kubectl create -f /vagrant/smoke-test.yaml"
+    controlplane.vm.provision :shell, privileged: false, inline: "kubectl create -f /var/sync/linux/smoke-test.yaml"
 
```

Comment:
I think you forgot to add the the `smoke-test.yaml` to `sync/linux/`

Reply:
fixed